src/test/regress/expected/multi_partition_pruning.out

```diff
@@ -4,12 +4,12 @@
 -- Tests to verify that we correctly prune unreferenced shards. For this, we
 -- need to increase the logging verbosity of messages displayed on the client.
 SET client_min_messages TO DEBUG2;
-SELECT l_orderkey, l_linenumber, l_shipdate FROM lineitem WHERE l_orderkey = 9030;
+-- Adding additional l_orderkey = 1 to make this query not router executable
```

Comment:
Minor: I feel like that instead of `WHERE l_orderkey = 9030 or l_orderkey = 1;` we should have `WHERE l_orderkey = 9030 or l_orderkey = 9030;` so that the outputs/shards not change?